app/index.html

```diff
@@ -177,10 +178,10 @@ <h2 class="page-title">Saved</h2>
 
             <section data-route="tuning">
               <paper-material elevation="1" id='background'>
-                <h2 class="page-title">Tuning</h2>
                 <my-tuner></my-tuner>
-                <p><canvas id='tunerCanvas'
-                style='border: 1px solid rgb(170, 170, 170);left: 10px;top: 10px;width: 50%; -moz-user-select: none;'></canvas></p>
+                <p><canvas id='tunerCanvas' width="500" height="225" 
+                style='border: 1px solid rgb(170, 170, 170);left: 10px;top: 10px;width: 60%; float: right -moz-user-select: none;'></canvas>
```

Comment:
Get the styling out of the structure, put it in the styles/ dir

you can use https://github.com/Rhernandez513/handsonhelping as a quick-and-dirty ref guide